scripts/mindmup_as_attack_trees.py

```diff
@@ -423,7 +423,25 @@ def update_node_apt(node, apt):
 	if node.get('attr', None) is None:
 		node.update({'attr': dict()})
 
-	node.get('attr').update({'evita_apt': apt})
```

Comment:
oh, in multiple places...

the .py files are now setup to import common functions from this .py file. c.f. ` from mindmup_as_attack_trees import *` in the `add-evita.py`.

So you should define this color LUT as a function in this module and call it from the callsite of `.update()` referencing `apt_colour` below and also from the analog callsite in the `add-evita.py` script